driver/src/main/com/mongodb/DBCollection.java

```diff
@@ -832,6 +833,69 @@ public DBObject findOne(final DBObject query, final DBCollectionFindOptions find
     }
 
     /**
+     * Creates a change stream for this collection.
+     *
+     * @return the change stream iterable
+     * @mongodb.driver.manual reference/operator/aggregation/changeStream $changeStream
+     * @since 3.6
+     */
+    Cursor watch() {
+        return watch(new DBCollectionChangeStreamOptions());
+    }
+
+    /**
+     * Creates a change stream for this collection.
+     *
+     * @param changeStreamOptions the options for the change stream.
+     * @return the change stream iterable
+     * @mongodb.driver.manual reference/operator/aggregation/changeStream $changeStream
+     * @since 3.6
+     */
+    Cursor watch(final DBCollectionChangeStreamOptions changeStreamOptions) {
+        return watch(Collections.<DBObject>emptyList(), changeStreamOptions);
+    }
+
+    /**
+     * Creates a change stream for this collection.
+     *
+     * @param pipeline the aggregation pipeline to apply to the change stream.
+     * @return the change stream iterable
+     * @mongodb.driver.manual reference/operator/aggregation/changeStream $changeStream
+     * @since 3.6
+     */
+    Cursor watch(final List<? extends DBObject> pipeline) {
+        return watch(pipeline, new DBCollectionChangeStreamOptions());
+    }
+
+    /**
+     * Creates a change stream for this collection.
+     *
+     * @param pipeline the aggregation pipeline to apply to the change stream.
+     * @param changeStreamOptions the options for the change stream.
+     * @return the change stream iterable
+     * @mongodb.driver.manual reference/operator/aggregation/changeStream $changeStream
+     * @since 3.6
+     */
+    Cursor watch(final List<? extends DBObject> pipeline, final DBCollectionChangeStreamOptions changeStreamOptions) {
+        notNull("pipeline", pipeline);
+        notNull("changeStreamOptions", changeStreamOptions);
+
+        DBCollectionChangeStreamOptions options = changeStreamOptions.copy();
```

Comment:
Why is copy needed?  It shouldn't be since the copy is neither modified nor stored for future use.